src/main/scala/io/opentargets/etl/backend/Target.scala

```diff
@@ -27,16 +27,21 @@ object TargetHelpers {
 
     }
 
-    def transformReactome: DataFrame = {
+    def transformReactome(reactomeDF: DataFrame): DataFrame = {
+
+      val reactomeIds = reactomeDF.select("id").collect().map(_(0)).toSeq.map(lit(_))
```

Comment:
@cmalangone brute force but needed this time. Let's remind ourselves to remove for the next release

Reply:
The new target will manage the relationship between target and reactome. 
@JarrodBaker tomorrow I will give you more details about this piece of code and why we need it for this release